src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -59,15 +61,17 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
   }
 
   async checkAudioOutput(
-    audioOutputDeviceInfo: MediaDeviceInfo,
+    audioOutputDeviceInfo: MediaDeviceInfo | string,
     audioOutputVerificationCallback: () => Promise<boolean>,
     audioElement: HTMLAudioElement = null
   ): Promise<CheckAudioOutputFeedback> {
     try {
-      const audioOutputDeviceId =
-        audioOutputDeviceInfo && audioOutputDeviceInfo.deviceId
-          ? audioOutputDeviceInfo.deviceId
-          : '';
+      let audioOutputDeviceId = '';
+      if (typeof audioOutputDeviceInfo === 'string') {
+        audioOutputDeviceId = audioOutputDeviceInfo;
+      } else if (audioOutputDeviceInfo && audioOutputDeviceInfo.deviceId) {
```

Comment:
Extract `DefaultDeviceController.getIntrinsicDeviceIdStr` and use that instead.

Reply:
Good call. Will do it :)